common/workunit/workunit.hpp

```diff
@@ -1688,8 +1690,10 @@ extern WORKUNIT_API bool isValidPriorityValue(const char * priority);
 extern WORKUNIT_API bool isValidMemoryValue(const char * memoryUnit);
 
 #define HourToSeconds(n) ((n)/3600)
```

Comment:
On reflection this should be an inline function, not a macro.  It is also confusing - why does converting from an hour to seconds divide by 3600?  I would expect the opposite.

Reply:
I've named this poorly.  It was designed to convert from hourly rate to per second rate.